judger-v2/Program.cpp

```diff
@@ -201,21 +194,9 @@ int Program::Compile(string source, int language) {
         system(("kill `pstree -p "+Inttostring(cpid)+" | sed 's/(/\\n(/g' | grep '(' | sed 's/(\\(.*\\)).*/\\1/' | tr \"\\n\" \" \"`").c_str());
```

Comment:
consider setpgid in previous section and kill by group. Same below. But this could be a separate task.